pkg/model/gcemodel/firewall.go

```diff
@@ -63,11 +64,15 @@ func (b *FirewallModelBuilder) Build(c *fi.ModelBuilderContext) error {
 		// The traffic is not recognized if it's on the overlay network?
 		klog.Warningf("Adding overlay network for X -> node rule - HACK")
 
+		nonMasqueradeCIDR, err := cidr.NewSet([]string{b.Cluster.Spec.NonMasqueradeCIDR})
+		if err != nil {
+			return err
+		}
 		t := &gcetasks.FirewallRule{
 			Name:         s(b.SafeObjectName("cidr-to-node")),
 			Lifecycle:    b.Lifecycle,
 			Network:      b.LinkToNetwork(),
-			SourceRanges: []string{b.Cluster.Spec.NonMasqueradeCIDR},
+			SourceRanges: ipv4SourceRange(nonMasqueradeCIDR),
 			TargetTags:   []string{b.GCETagForRole(kops.InstanceGroupRoleNode)},
 			Allowed:      []string{"tcp", "udp", "icmp", "esp", "ah", "sctp"},
 		}
```

Comment:
Why aren't we building IPv6 firewall rules here?

Reply:
It _was_ because they weren't set by default in `kops create`.  However, now we're using a helper function, we do now build the rules for ipv4 and ipv6.